Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
```

Comment:
ok, I asked the author for a release, maybe we're lucky and he does not know about the policy ;)